interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.c

```diff
@@ -873,7 +869,17 @@ int acados_create()
 
     external_function_param_casadi_create(&r_e_cost, {{ dims.np }});
     {%- endif %}
-	for (int i = 0; i < N; ++i)
+
+    /* nlp_in */
+    nlp_in = ocp_nlp_in_create(nlp_config, nlp_dims);
+
+    double Ts = Tf/N;
```

Comment:
The mex interface also supports non-uniform multiple-shooting grids. It may be worthwhile to add this feature to the template interface already now, instead of leaving a "hidden" mismatch.

Reply:
Or if too complicated, at least add a note that this is only valid for uniform multiple-shooting grids, TODO the other, and so on...

Reply:
sure, done